src/documentation/0012-node-cli-args/index.md

```diff
@@ -78,6 +78,12 @@ const args = require('minimist')(process.argv.slice(2))
 args['name'] //joe
 ```
 
+Install the required `minimist` package using `npm` (lesson about the package manager comes [later on](https://nodejs.dev/learn/an-introduction-to-the-npm-package-manager)).
+
+```js
```

Comment:
Should be bash not js

Reply:
Hello, I just pushed an update for this. Thank your for your comment !